.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
.gitIgnore file is super useful to prevent us from committing local information to a git repo